lib/orbital/ext/active_merchant/active_merchant.rb

```diff
@@ -169,6 +171,14 @@ def build_new_order_xml_with_cc(operation, money, creditcard, options)
         end
       end
 
+      def add_mit_cit_params(xml, options)
+        xml.tag! :MITMsgType, options[:mit_cit_type] unless options[:mit_cit_type].nil?
```

Comment:
Could you document in the `README` how `mit_cit_type` and `mit_ref_trx_id` should be populated?

Also, if this is for VISA only, is it the responsibility of the control plugin to filter on the card type?

Reply:
Yes, added the README section. 

Correct, it is the control plugin's responsibility to decide if these params should be sent or not.